### DISCUSSION TITLE
.net 5 UploadImage

### DISCUSSION THREAD

**Discussion by inproctecnologia** [serenity-is/Serenity#5628] [Help]
Hi Everybody.

I migrated an Asp.Net application to Serenity and upgraded the core to the latest stable version released in January 2021. This application of mine ran on an architecture of 3 distinct servers (DB, APP and Storage Files). Using Serenity, the files that needed to be accessed in the storage server are accessed through common network paths (\\\\servername\sharing\folder\sfiles....). I am unable to map in UploadEditor this folder and continue saving files on the network because when I click on thumbs it calls http://domain/upload//network//sharing//folder//files...
Can you help me?

**Comment by dfaruque** [comment]
you need to set upload path in the appSettings.json

```
  "UploadSettings": {
    "Path": "//fileservername/App_Uploads" //put your network path here
  }
```
